e2e/cypress/support/ui_commands.js

```diff
@@ -374,6 +374,27 @@ Cypress.Commands.add('updateChannelHeader', (text) => {
         wait(TIMEOUTS.TINY);
 });
 
+/**
+ * On default "ad-1" team, create and visit a new channel
+ */
+Cypress.Commands.add('createAndVisitNewChannel', () => {
+    cy.visit('/ad-1/channels/town-square');
+
+    cy.getCurrentTeamId().then((teamId) => {
+        cy.apiCreateChannel(teamId, 'channel-test', 'Channel Test').then((res) => {
+            const channel = res.body;
+
+            // # Visit the new channel
+            cy.visit(`/ad-1/channels/${channel.name}`);
+
+            // * Verify channel's display name
+            cy.get('#channelHeaderTitle').should('contain', channel.display_name);
+
+            cy.wrap({channel});
```

Comment:
Is channel already an object? I think `cy.wrap(channel)` would be sufficient. If not, everything else looks good!

Reply:
Yup, an object. That works, too. Will change.

Reply:
Updated, PTAL